federated_learning/nvflare/nvflare_example/data/Task09_Spleen/dataset_8.json

```diff
@@ -0,0 +1,46 @@
+{
+	"training": [{
```

Comment:
why put duplicate files in `nvflare_example/data/` and `nvflare_example/data/Task09_Spleen`

Reply:
Right. The folder `nvflare_example/data/Task09_Spleen` was committed by mistake. I'll remove it.
It's necessary to copy the data lists into Task09_Spleen in [download_dataset.sh](https://github.com/holgerroth/tutorials/blob/11478a4d8a5a3161962fd042a07b595dafaa285b/federated_learning/nvflare/nvflare_example/data/download_dataset.sh#L6) to make the train_configer.py work which requires the data list inside the `data_list_base_dir`. However, I cannot provide the lists there because `monai.apps.utils.download_and_extract` requires an empty target directory to work... Hence the lists are in `nvflare_example/data` and are only copied to `Task09_Spleen` after extraction.